StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Vertical Main Screen/RowButton.swift

```diff
@@ -393,7 +416,7 @@ extension RowButton {
             }
             return paymentMethodType.displayName
         }()
-        return RowButton(appearance: appearance, imageView: imageView, text: text, subtext: subtitle, promoText: promoText, rightAccessoryView: rightAccessoryView, shouldAnimateOnPress: shouldAnimateOnPress, isEmbedded: isEmbedded, didTap: didTap)
+        return RowButton(appearance: appearance, originalCornerRadius: originalCornerRadius, imageView: imageView, text: text, subtext: subtitle, promoText: promoText, rightAccessoryView: rightAccessoryView, shouldAnimateOnPress: shouldAnimateOnPress, isEmbedded: isEmbedded, didTap: didTap)
```

Comment:
Doesn't the value of `originalCornerRadius` come from `appearance`? Do we need to pass this property in here alongside `appearance`, or is it redundant? Could be missing something here though!

Reply:
The call site is a bit different:

```swift
let rowButton = RowButton.makeForPaymentMethodType(
  // …
  appearance: rowButtonAppearance,
  originalCornerRadius: appearance.cornerRadius
)
```

`rowButtonAppearance` is derived from `appearance`, but with corner radiuses removed. Maybe @porter-stripe has an idea of how to make this a little cleaner?